app/uk/gov/nationalarchives/omega/editorial/views/editSetRecordEdit.scala.html

```diff
@@ -196,20 +197,20 @@
         role = Some("group")
     ))
 @govukSelect(Select(
-    id = "legalStatus",
-    name = "legalStatus",
+    id = FieldNames.legalStatus,
+    name = FieldNames.legalStatus,
     items = legalStatusItems ++ legalStatusReferenceData.map { status =>
         SelectItem(
-            selected = editSetRecordForm.data.get("legalStatus").contains(status.uri),
+            selected = editSetRecordForm.data.get(FieldNames.legalStatus).contains(status.uri),
             text = status.label,
             value = Some(status.uri)
         )
     },
-    label = Label(forAttr = Some("legalStatus"), content = Text(messages("edit-set.record.edit.legal-status"))),
-    errorMessage = editSetRecordForm.error("legalStatus").map { error =>
+    label = Label(forAttr = Some(FieldNames.legalStatus), content = Text(messages("edit-set.record.edit.legal-status"))),
+    errorMessage = editSetRecordForm.error(FieldNames.legalStatus).map { error =>
         ErrorMessage(
             content = Text(error.message),
-            attributes = Map("for" -> "legalStatus")
+            attributes = Map("for" -> FieldNames.legalStatus)
         )
     },
 )
```

Comment:
```
@govukTextarea(Textarea(
    id = "custodial-history",
    name = "custodial-history",
    value = editSetRecordForm.data.get("custodial-history"),
 
```
@felixgb does this need to be replaced by FieldNames.custodialHistory, appears it might have been missed